Common/include/interpolation_structure.hpp

```diff
@@ -253,5 +285,98 @@ class CMirror : public CInterpolator {
    * \param[in] config - Definition of the particular problem.
    */
   void Set_TransferCoeff(CConfig **config);
+  
+};
+
+/*!
+ * \brief Sliding mesh approach
+  */
+class CSlidingMesh : public CInterpolator {
+public:
```

Comment:
OK, I do see an inconsistency in the name that can be misleading as this class is a specific interpolation method that can be used also with sliding meshes, but not the only one. I agree, if you prefer we can change this later on.